packages/automerge-repo/test/CollectionSynchronizer.test.ts

```diff
@@ -45,9 +45,7 @@ describe("CollectionSynchronizer", () => {
   it("should not synchronize to a peer which is excluded from the share policy", done => {
     const handle = collection.create()
 
-    collection.sharePolicy = async (peerId: PeerId, documentId: string) => {
-      return peerId !== "peer1"
-    }
+    collection.sharePolicy = async (peerId: PeerId) => peerId !== "peer1"
```

Comment:
If there's no per-document sharing logic, we don't require the share policy function to take a `documentId` parameter.